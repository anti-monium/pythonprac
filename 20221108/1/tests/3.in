a = DivStr('lolkekcheburek')
b = DivStr('keklolorbidol')
print(*(a % 8 + b % 10) // 4)
print(*(a + b + a) // 17)
print((a * 2 + b * 4) % 22)
