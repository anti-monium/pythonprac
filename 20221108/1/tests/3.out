eb ur ek do
lo lk ek ch eb ur ek ke kl ol or bi do ll ol ke kc
lkeklolorbidol
